src/showcase/Showcase.story.mdx

```diff
@@ -0,0 +1,34 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs/blocks'
+
+import { Paragraph } from '../paragraph'
+import { Showcase } from './index'
+
+<Meta title="Widgets/Showcase" />
+
+# Showcase
+
+<Canvas>
+  <Story name="Default">
+    <Showcase>
+      <div>Lists and loops</div>
+      <div>through all children</div>
+      <div>regardless of their number</div>
+      <div>or their type</div>
+      <Paragraph>(here's a Paragraph)</Paragraph>
+    </Showcase>
+  </Story>
+</Canvas>
+
+## Specifications
+
```

Comment:
Missing specs.. 
```suggestion
Showcase is **marquee** component
```

Reply:
Added a line, but we don't really have specs for this.